saltstack/pillar/custom/release_standalone.sls

```diff
@@ -1,12 +1,12 @@
 {# Custom Settings #}
 
 {% set branch = 'release' %} {# Use 'release' or 'devel' #}
-{% set version = '3.18' %}
+{% set version = '3.19' %}
 {% set environment = 'dev' %} {# Use 'dev' or 'prod' #}
-{% set r_download = 'https://cran.r-project.org/src/base/R-4/R-4.3.3.tar.gz' %}
-{% set r_version = 'R-4.3' %}
-{% set r_previous_version = 'R-4.2' %}
-{% set cycle = 'patch' %} {# Use 'devel' for Spring to Fall, 'patch' for Fall to Spring #}
+{% set r_download = 'https://cran.r-project.org/src/base/R-4/R-4.4.0.tar.gz' %}
+{% set r_version = 'R-4.4' %}
+{% set r_previous_version = 'R-4.3' %}
+{% set cycle = 'devel' %} {# Use 'devel' for Spring to Fall, 'patch' for Fall to Spring #}
```

Comment:
This should be `patched`. I had assumption that this won't be changed because we never usually make a "release" machine; we always configure a devel machine. I'm going to remove this in the future.